python/interpret_community/dataset/dataset_wrapper.py

```diff
@@ -151,22 +151,42 @@ def typed_dataset(self):
         wrapper_func = self.typed_wrapper_func
         return wrapper_func(self._dataset)
 
-    def typed_wrapper_func(self, dataset):
+    @property
+    def typed_dataset_without_index(self):
```

Comment:
Another one? We really need to clean this thing up

Reply:
actually it's only used in one place.  I got rid of it.